test/TestProjects/MgmtListMethods/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -125,15 +125,15 @@ private static QuotasRestOperations GetQuotasRestOperations(ClientDiagnostics cl
         /// <param name="subscription"> The <see cref="Subscription" /> instance the method will execute against. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <returns> A collection of resource operations that may take multiple service requests to iterate over. </returns>
-        public static AsyncPageable<FakeParentWithAncestorWithNonResChWithLocData> GetBySubscriptionFakeParentWithAncestorWithNonResChWithLocsAsync(this Subscription subscription, CancellationToken cancellationToken = default)
+        public static AsyncPageable<FakeParentWithAncestorWithNonResChWithLocData> ListFakeParentWithAncestorWithNonResChWithLocsAsync(this Subscription subscription, CancellationToken cancellationToken = default)
```

Comment:
Not the latest generated code?

Reply:
I am not sure. I will take a look at this after yours is merged